bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigResolver.java

```diff
@@ -163,22 +158,14 @@ public ConfigResolver(Module rootModule, Map<Module, VariableKey[]> configVarMap
 
         // Handle errors while getting config values.
         if (configValue.isPresent()) {
-            exceptionList.forEach(e -> diagnosticLog.warn(e.getMessage()));
+            exceptionList.forEach(e -> diagnosticLog.warn(e.getErrorCode(), key.location, e.getArgs()));
             return configValue;
         }
         if (exceptionList.isEmpty() && key.isRequired) {
-            String displayName;
```

Comment:
Since the variable location already contains module data, we do not need the module name in the error message content.